helm/cas-provision/templates/djangoSuperUser.yaml

```diff
@@ -0,0 +1,13 @@
+{{- if hasPrefix .Values.namespacePrefixes.obps .Release.Namespace}}
+kind: Secret
+apiVersion: v1
+metadata:
+  name: django-superuser
+  labels: {{ include "cas-provision.labels" . | nindent 4 }}
+type: Opaque
+stringData:
+{{- if hasSuffix "-dev" .Release.Namespace }}
```

Comment:
This could go on line 1 & then we won't create the secret at all in test/prod/tools. I think this will currently create an empty secret in those namespaces?